packages/shared/src/lib/featureManagement.ts

```diff
@@ -16,7 +16,7 @@ const feature = {
   feedVersion: new Feature('feed_version', 15),
   search: new Feature('search', SearchExperiment.Control),
   lowImps: new Feature('feed_low_imps'),
-  bookmarkOnCard: new Feature('bookmark_on_card', false),
+  bookmarkOnCard: new Feature('bookmark_on_card', true),
```

Comment:
Just a reminder to revert this before merge.

Reply:
Oh no 🤦